docs/photutils/psf.rst

```diff
@@ -481,6 +481,159 @@ Consider the previous example after the line
+        --- -------- ------------- ------------- ------------- -------------
+          1        1             1 17.9830836988  17.014907321 9753.12898388
+          1        1             2 12.6296930472 19.6647630336 722.116920665
+    >>> phot_results_itr['sigma_0', 'sigma_fit', 'x_fit', 'y_fit', 'flux_fit'] #doctest: +SKIP
+        sigma_0   sigma_fit       x_fit         y_fit        flux_fit
+        ------- ------------- ------------- ------------- -------------
+           2.05 2.03898434182 17.9155310586 17.0377322599 10478.1845716
+           2.05 1.77365765325 12.2809666868 19.2416219146 772.882624378
+
+We can see that ``sigma_0`` (the initial guess for ``sigma``) was assigned
+to the value we used when creating the PSF model.
+
+Let's take a look at the residual image::
+
+    >>> plt.imshow(itr_phot_obj.get_residual_image(), cmap='viridis',
+    ... aspect=1, interpolation='nearest', origin='lower') #doctest: +SKIP
+
+.. plot::
+
+    from photutils.datasets import make_random_gaussians, make_noise_image
+    from photutils.datasets import make_gaussian_sources
+    import matplotlib.pyplot as plt
+    from photutils.psf import IterativelySubtractedPSFPhotometry, BasicPSFPhotometry
+    from astropy.stats import gaussian_sigma_to_fwhm
+    from astropy.table import Table
+    from photutils import MMMBackground
+    from photutils.psf import IntegratedGaussianPRF, DAOGroup
+    from photutils.detection import DAOStarFinder
+    from astropy.modeling.fitting import LevMarLSQFitter
+    from photutils.detection import IRAFStarFinder
+
+    sources = Table()
+    sources['flux'] = [10000, 1000]
+    sources['x_mean'] = [18, 13]
+    sources['y_mean'] = [17, 19]
+    sources['x_stddev'] = [2] * 2
+    sources['y_stddev'] = sources['x_stddev']
+    sources['theta'] = [0] * 2
+    tshape = (32, 32)
+    image = (make_gaussian_sources(tshape, sources) +
+             make_noise_image(tshape, type='poisson', mean=6.,
+                              random_state=1) +
+             make_noise_image(tshape, type='gaussian', mean=0.,
+                              stddev=2., random_state=1))
+
+    daogroup = DAOGroup(crit_separation=8)
+    mmm_bkg = MMMBackground()
+    psf_model = IntegratedGaussianPRF(sigma=2.05)
+    iraffind = IRAFStarFinder(threshold=2.5*mmm_bkg(image),
+                              fwhm=4.5)
+    fitter = LevMarLSQFitter()
+    psf_model.sigma.fixed = False
+
+    itr_phot_obj = IterativelySubtractedPSFPhotometry(finder=iraffind, group_maker=daogroup,
+                                        bkg_estimator=mmm_bkg, psf_model=psf_model,
+                                        fitter=fitter, fitshape=(11, 11), niters=2)
+
+    phot_results_itr = itr_phot_obj(image)
+    plt.imshow(itr_phot_obj.get_residual_image(), cmap='viridis', aspect=1,
+            interpolation='nearest', origin='lower')
+
+
 For more examples, also check the online notebook in the next section.
```

Comment:
Because the above text has gotten a lot longer, maybe we should just drop this "For more examples" sentence?  Right now it looks like its referring to the "Fitting additional paramters" functionality, which of course isn't actually true.  And maybe change the next section to "Additional example notebooks (online)"?